lib/checks/canMultiplyLikeTermPolynomialNodes.js

```diff
@@ -13,7 +13,6 @@ function canMultiplyLikeTermPolynomialNodes(node) {
   if (args.length === 1) {
     return false;
   }
-
```

Comment:
careful with these extra lines added/removed - if you scan through the diff you should be able to see them all without me having to comment